src/Nncase.Core/ValueRange.cs

```diff
@@ -23,6 +23,11 @@ public record struct ValueRange<T>(T Min, T Max)
 
     public bool IsFull => this == Full;
 
+    /// <summary>
+    /// Gets translate ValueRange to tensor{Min, Max}.
+    /// </summary>
+    public Tensor ToTensor => Tensor.FromArray(new[] { Min, Max });
```

Comment:
It's weird to add this property:
- Not a good naming, users may think it equals to `Tensor.FromScalar(this). A good naming should let users know what kind of operations this method does.
- Use property getter to construct a new object is [not recommended](https://stackoverflow.com/a/2103719)
- There is not an obvious reason to add this getter to this datatype's definition, it's better to define an extension method